definitions/divorce/json/State/State-deemed-and-dispensed-nonprod.json

```diff
@@ -22,5 +22,21 @@
     "Name": "Service Application Not Approved",
     "Description": "Service Application Not Approved",
     "DisplayOrder": 34
+  },
+  {
+    "LiveFrom": "23/07/2020",
```

Comment:
please remove it.

It's been redundant state that has been removed already
`AwaitingDNApplication`

Reply:
`AwaitingDecreeNisi` should be used instead. (And is already used in COS and CCD master).

Reply:
removed